hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneAddress.java

```diff
@@ -47,26 +48,34 @@ public static Collection<Object[]> data() {
     });
   }
 
-  private String prefix;
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
+
+  private final String prefix;
+  private OzoneAddress address;
 
   public TestOzoneAddress(String prefix) {
     this.prefix = prefix;
   }
 
   @Test
-  public void checkUrlTypes() throws OzoneClientException, IOException {
-    OzoneAddress address;
-
+  public void checkRootUrlType() throws OzoneClientException {
     address = new OzoneAddress("");
```

Comment:
What's the difference between this and last parameter in line 47?

Reply:
There is no difference, check [here](https://github.com/apache/ozone/blob/a20e84ef03517d8970d8094cd4ec85d3d53caee2/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/OzoneAddress.java#L70), but we need this call to initialize `OzoneAddress`, even if we are setting an empty address.

Reply:
I still don't get to have empty string in test cases. It is unnecessary to me. But I'll leave it because it was already there.